scripts/rebase-helper.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+
+# This script is meant to be used to help out when performing a rebase.
+# Particularly, when two people have editted the templates, the generated file
+# needs to be updated based on the merge of your parts edits. This script attempts
+# to generate, add the templates.go file, fix any of the expected testdata output,
+# and then attempts to push the rebase forward.
+
+# If anything fails, we bail out and leave the user to repair.
+
+# This will fail to push the rebase forward if there are other conflicts to be resolved.
+# It's recommended you re-run this script again after resolving them to ensure that the
+# generated file and expected test output reflects your merge resolutions.
+
+go generate -v ./...
+cd pkg/acsengine/testdata
+mv **/*.json{.err,}
```

Comment:
It's "globbing".

It will (or rather, should - testing is in progress) expand to:
```
$ mv foo/bar.json.err foo/bar.json
$ mv whatever/something/else/cool.json.err whatever/something/else/cool.json
etc...
```